docs/installing.adoc

```diff
@@ -277,6 +278,10 @@ host-based tools.
 
 === Manually Download the Vagrant Box
 
+=======
+* Manually Downloading the Vagrant Box Image
++
+>>>>>>> b60c866... Fix #566 Updated Proxy section with the relevant steps from VSM.
```

Comment:
@Preeticp Looks like the rebase did not work properly for you. Please clean up the PR.